zerver/tests/test_user_groups.py

```diff
@@ -706,7 +706,7 @@ def do_test_set_group_setting_during_user_group_creation(self, setting_name: str
             "members": orjson.dumps([hamlet.id]).decode(),
             "description": "Frontend team",
         }
-        params[setting_name] = orjson.dumps(1111).decode()
+        params[setting_name] = orjson.dumps(-1).decode()
```

Comment:
A key detail here is that group IDs keep increasing across a test suite run, so every realm we create in a test adds 10 or whatever even after the transaction reverts, I think.